dashboards/mobile/mobile.json

```diff
@@ -0,0 +1,308 @@
+                {
+                  "accountId": 0,
+                  "query": "SELECT uniqueCount(uuid) as 'Installed devices' FROM MobileUserAction SINCE 1 month ago COMPARE WITH 1 month ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": true
+              }
+            }
+          },
+          {
+            "title": "Installations by country",
+            "layout": {
+              "column": 5,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.pie"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": true
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT uniqueCount(uuid) FROM MobileRequest FACET `countryCode` SINCE 1 month ago EXTRAPOLATE"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": true
+              }
+            }
+          },
+          {
+            "title": "App Version",
+            "layout": {
+              "column": 9,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.pie"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": true
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT uniqueCount(appVersion) FROM MobileRequest FACET appVersion SINCE 1 month ago EXTRAPOLATE"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "App launches",
+            "layout": {
+              "column": 1,
+              "row": 4,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.area"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT count(newrelic.timeslice.value) FROM Metric WHERE (entityGuid = 'MzU5NTY5NHxNT0JJTEV8QVBQTElDQVRJT058NTM1ODcyMjEx') AND (metricTimesliceName = 'Session/Start') FACET `instanceName` SINCE 30 minutes ago TIMESERIES"
```

Comment:
Entity GUIDs are unique to an account within New Relic, so if other people install this dashboard they will never get any results in this chart since their entity GUID will be different. Could you remove this WHERE clause?

Reply:
Done!